security/src/test/java/com/twilio/security/crypto/key/cipher/BiometricAuthenticatorTests.kt

```diff
@@ -0,0 +1,810 @@
+/*
+ * Copyright (c) 2021 Twilio Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import androidx.biometric.BiometricManager
```

Comment:
<table data-meta="generated_by_Linter">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-android/blob/838d47e1c0f4545de85ffc209bff32b430e234e2/security/src/test/java/com/twilio/security/crypto/key/cipher/BiometricAuthenticatorTests.kt#L17"></span>Imports must be ordered in lexicographic order without any empty lines in-between</td>
    </tr>
  </tbody>
</table>